utilize_eye_in_hand_calibration.py

```diff
@@ -0,0 +1,94 @@
+"""
+Utilize the result of eye-in-hand calibration to transform (picking) point
+coordinates from the camera frame to the robot base frame.
+"""
+
+from pathlib import Path
+import numpy as np
+import cv2
+
+
+def _assert_if_valid(file_name):
+    """
+    Check if YAML file is valid.
+
+    Args:
+        file_name: Path to YAML file.
+
+    Raises:
+        FileNotFoundError: Not able to open YAML file.
```

Comment:
You can be more specific here:
Take a look at this one's raises section: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
thanks for the link! done